Server/tsconfig.json

```diff
@@ -15,6 +15,7 @@
     "typeRoots": [
       "./node_modules/@types"
     ],
+    "resolveJsonModule": true,

```

Comment:
this is required because we want to import the json deploy script in order to validate it? @dailam-msft

Reply:
Yes that is right